Adds case study for Wordpress (#154)
NoroffFEU/development-platforms: src/content/docs/cms/wordpress-case-study.md

- **2006:** The first WordCamp was held in San Francisco *(WordCamps are annual events that volunteers organize for users and developers to get together and discuss all things WordPress).*
- **2007:** Automatic purchased Gravatar *(Gravatar are the avatars, or pictures you see next to your name when you interact on WordPress sites).* 
- **2008:** Launch of the WordPress Theme Directory
- **2009:** WordPress wins the Packt best Open Source CMS Awards.
- **2010:** Version 3.0, includes the feature for custom post types.
- **2011:** WordPress powers over 12% of the world’s websites.
- **2013:** WordPress admin interface becomes responsive.
- **2017:** User customizations can be saved as drafts or scheduled to be published in the future.

## Main Features

The open-source content management system, WordPress, is at its core, a tool that makes it easy for developers and non-developers to make and manage important aspects of websites (e.g. content) without knowing much or anything about programming. This means that building a website can be accessible to anyone, even people who aren’t developers.

7 good reasons to use WordPress:

**1. Free and open-source software**

> WordPress is free for anyone to download. And being an open-source software means that WodPress is developed by a collection of contributers, but fear not, you don’t have to contribute by e.g. being a beta tester or a brand advocate if you don’t want to. 

Reviewer: ```suggestion
WordPress is free for anyone to download. And being open-source software it means that WordPress is developed by a collection of contributors. But fear not, you don’t have to contribute (by e.g. being a beta tester or a brand advocate ) if you don’t want to. 
```

**Note** that even though the software is free, you will most likely end up spending a little bit of money. WordPress is self-hosted so you will need hosting services.

**2. It’s adaptable**

When WordPress first was released it was originally created as a tool to publish blogs, but with the variaty of new releases over the years the platform has changed. Now, with the adaptions made to the software you can easily make pretty much anything you can think of with WordPress. 

Here are a few examples:
- Blogs
- eCommerce sites
- Business sites
- Chatrooms
- Non-profit websites (i.e. for collecting donations)
- Portfolios
- Job boards
- Forums

With WordPress you can easily make forums and eCommerce sites that function with simple plugins and themes.

**Note:** Some plugins and themes require a small one-time fee.

**3. Supports various media types**

If you want to put a photo, gif, video or document on your website, WordPress supports most file types. 

**Note:** Make sure to follow the rules and use legally media that is either owned by you, open for free download or usable when credits are given (this goes for all websites, not just WordPress).

**4. Easy to Learn**

> The WordPress software is easy to use, anoyone could start playing around with the dashboard and then ten minutes in you’d have an understanding of the interface. There‘s also tons of blogs, forums, online courses, seminars, webinars, books etc. outlining the different aspects of the platform.  

Reviewer: ```suggestion
The WordPress software is easy to use, anyone can start playing around with the dashboard and then ten minutes in you’d have an understanding of the interface. There are also tons of blogs, forums, online courses, seminars, webinars, books, etc outlining the different aspects of the platform.  
```

## Market Comparison

| WordPress | Squarespace |
| --- | --- |
| Customizable with plugins, themes, REST API and third-party apps | "Built-in" features and integrations |
| Free open-source CMS | Subscription all-in-one CMS |
| The user is responsible for all maintenance | "Closed" environment = the platform manages all the updates and maintenance |
| Big community with resources and tutorials | Dedicated support team with organized tutorials |
| Editing in the backend | Drag-and-drop (“what you see is what you get”) editing |

## Getting Started

The first step is to buy a domain name and a hosting plan. A domain functions as your website’s address online, whereas web hosting is responsible for storing your website files.

On [WordPress](https://wordpress.org/download/) users can install the software manually or a lot of hosting services have automatic installation. Create your WordPress login details and connect your custom domain to the site. Once everything is done you can log in to the WordPress dashboard and start building your website.

## Conclusion

> WordPress is a powerful and popular software available for both developers and non-developers who want to be able to fully customize their website/platform and have complete control on every aspect of their project. 

Reviewer: ```suggestion
WordPress is a powerful and popular software available for both developers and non-developers who want to be able to fully customize their website/platform and have complete control over every aspect of their project. 
```
